src/event.rs

```diff
@@ -158,3 +158,13 @@ pub enum EventType<TYPES: NodeType> {
         sender: TYPES::SignatureKey,
     },
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub enum HotShotAction {
```

Comment:
Should we also include QC?

Reply:
I don't think that action can be attributable by other nodes.